plugins/grpc-ecosystem/gateway/v2.15.0/buf.plugin.yaml

```diff
@@ -0,0 +1,12 @@
+version: v1
+name: buf.build/grpc-ecosystem/gateway
```

Comment:
Okay, I had second thoughts on this, but because buf.build/grpc-ecosystem/grpc-gateway already exists as a module, we'll need to name this something else.